index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="flex-container">
+           <h1 class="flex-item">Welcome</h1>
+           <a href="#"><img class="flex-item" src="./images/Screenshot_1.jpg" alt=""></a>
+        </nav>
+    </header>
+    <section>
+        <h2>Hey there. I’m Mariane</br>
```

Comment:
- [ ] This title should be tagged with `h1`. Kindly fix it. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info

Reply:
Fixed